devito/types/sparse.py

```diff
@@ -47,9 +47,10 @@ def __init_finalize__(self, *args, **kwargs):
     def __indices_setup__(cls, **kwargs):
         dimensions = kwargs.get('dimensions')
         if dimensions is not None:
-            return dimensions
+            return dimensions, dimensions
```

Comment:
Same arg twice?

Reply:
that's on purpose, other subclasses will act differently